onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportProblemFragmentBase.java

```diff
@@ -237,4 +220,9 @@ public ObaResponse loadInBackground() {
     protected abstract int getLayoutId();
 
     protected abstract ReportLoader createLoader(Bundle args);
+
+    public void setCallback(ReportProblemFragmentCallback callback) {
+        this.mCallback = callback;
```

Comment:
Sorry, to clarify - you should be able to change this line to:
`mCallback = callback;`

So, just remove the reference to "this" instance of the class.